provision/acc_provision/cloud_provision.py

```diff
@@ -169,12 +169,13 @@ def Run(self, flavor_opts, kube_yaml_gen_func):
         output_tar = self.args.output_tar
         operator_cr_output_file = self.args.aci_operator_cr
         gen(self.config, output_file, output_tar, operator_cr_output_file)
-        if self.args.flavor == "aks":
-            self.print_aks_setup()
-        elif self.args.flavor == "eks":
-            self.print_eks_setup()
-        else:
-            self.print_openshift_setup()
+        if "test_run" in self.args and not self.args.test_run:
```

Comment:
this still does not work, because now when you actually run acc-provision the files are never going to get generated!

I still dont understand why the earlier suggestion did not work:

    if not "test_run" in self.args:

Reply:
hmm okay, 
the earlier suggestion as suggested by you which can be written in 3 ways in python syntax which all mean same -
if "test_run" not in self.args: , if not ("test_run" in self.args): , if not "test_run" in self.args:
for all of these 3 I tried, the acirc file get's generated but I couldn't logically understand why it's getting generated though.